lessons/projects/pong/index.md

```diff
@@ -0,0 +1,579 @@
+# Pong
+
+<div style="text-align:center;">
+    <img src="{{ static('pong.png') }}" alt="">
+</div>
+
+Dnes si prohloubíme znalosti programování grafických aplikací,
+které jsme získali na [lekci o Pygletu]({{ lesson_url('intro/pyglet') }}),
+na reálném problému.
+
+Naprogramujeme si s pomocí knihovny Pyglet
+jednu z prvních videoher, [Pong](https://en.wikipedia.org/wiki/Pong).
+Pong vydala společnost [Atari](https://en.wikipedia.org/wiki/Atari,_Inc.)
+jako svůj první titul v roce 1972 a odstartovala tak boom herního průmyslu.
+
+Na [Youtube](https://www.youtube.com/watch?v=fiShX2pTz9A)
```

Comment:
odkazem by v této větě mělo být lovo "video" nebo celé "video, které ukazuje jak se Pong hraje".